From 2f51f6b7478e69d8f9e8e3a31bddccf6af88c720 Mon Sep 17 00:00:00 2001
From: Pavel Moravec <pmoravec@redhat.com>
Date: Tue, 7 Nov 2017 14:28:33 +0100
Subject: [PATCH] [vdo] collect proper sys paths

VDO plugin shall collect /sys/kvdo and /sys/uds
instead of original /sys/vdo and /sys/albireo

Resolves: #1134

Signed-off-by: Pavel Moravec <pmoravec@redhat.com>
---
 sos/plugins/vdo.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sos/plugins/vdo.py b/sos/plugins/vdo.py
index e6eff4ad..bdf79886 100644
--- a/sos/plugins/vdo.py
+++ b/sos/plugins/vdo.py
@@ -27,8 +27,8 @@ class Vdo(Plugin, RedHatPlugin):
     profiles = ('storage',)
     packages = ('vdo',)
     files = (
-        '/sys/vdo',
-        '/sys/albireo',
+        '/sys/kvdo',
+        '/sys/uds',
         '/etc/vdoconf.yml',
         '/etc/vdoconf.xml'
     )
-- 
2.13.6

